docs/api/cases/cases-api-get-reporters.asciidoc

```diff
@@ -0,0 +1,60 @@
+[[cases-api-get-reporters]]
+== Get reporters API
+++++
+<titleabbrev>Get reporters</titleabbrev>
+++++
+
+Returns all case reporters (users who opened cases).
```

Comment:
> Returns all case reporters (users who opened cases).

Re-reading this (and not having personally tested the API), this description makes me wonder: is it only the users that specifically "opened" the case (i.e. changed the status to open), or is it also the users that *created* cases or maybe even the users that updated them?

Reply:
@lcawl Thanks for this suggestion. I tested the API and the response only contains the users who opened cases and does not contain users who updated the case, commented, or changed the status.